recipe/conda_build_config.yaml

```
@@ -0,0 +1,2 @@
+c_stdlib_version:
+  - "13"  # [osx and arm64]
```

URLs seem to be based on tags at https://github.com/alexey-lysiuk/macos-sdk/releases. Maybe try 13.3?

```suggestion
  - "13.3"  # [osx and arm64]
```